backout ab16ad33ba "patch 8.2.3941: SIGTSTP is not handled"
causes hangs on ^Z

see https://github.com/vim/vim/pull/9422

Index: runtime/doc/autocmd.txt
--- runtime/doc/autocmd.txt.orig
+++ runtime/doc/autocmd.txt
@@ -1285,8 +1285,8 @@ VimResume			When the Vim instance is resumed after bei
 	:autocmd VimResume * checktime
 <							*VimSuspend*
 VimSuspend			When the Vim instance is suspended.  Only when
-				CTRL-Z was typed inside Vim, or when the SIGTSTP
-				signal was sent to Vim, but not for SIGSTOP.
+				CTRL-Z was typed inside Vim, not when the
+				SIGSTOP or SIGTSTP signal was sent to Vim.
 							*WinClosed*
 WinClosed			After closing a window.  The pattern is
 				matched against the |window-ID|.  Both
